lib/api.js

```diff
@@ -8,6 +8,7 @@ const modelTasksRouter = require('./router/model-tasks');
 const decorators = require('./decorators');
 
 const profile = require('./middleware/profile');
+const resolveProjectVersionUpdates = require('./middleware/resolve-project-version-updates');
```

Comment:
Java called, it wants its function names back.

Reply:
when you say "Java", do you mean @wheelsandcogs ?

Reply:
:trollface:

Reply:
Nice function name 👍